### PR TITLE
Bump to 0.8

### PR DESCRIPTION
**Pull request by hoechenberger**
We needed to update some dependencies, so I din't wait for the bot.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -18,17 +18,18 @@ build:
...
   host:
-    - python >=3.6
+    - python >=3.7
     - pip
   run:
-    - python >=3.6
-    - numpy >=1.15.4
+    - python >=3.7
+    - numpy >=1.16.0
     - scipy >=1.5.0
```
Comment (sappelhoff) [inline recipe/meta.yaml:26]: scipy is stricly speaking >= 1.2.0, ... >=1.5.0 only for some EEGLAB things

**Comment by hoechenberger** [comment]
I specifically pinned to scipy >=1.5.0 so users can get all optional functionality. Do you disagree?

**Comment by sappelhoff** [comment]
I think it's a balance act between requiring scipy 1.5 (1 year old) vs 1.2 (2.5 years old) and making life easy for the users who want to use mne-bids with EEGLAB and don't want to run `conda update scipy` --> but overall I don't feel strongly about this, so if you want I can revert my commit :-)

just let me know

**Comment by sappelhoff** [comment]
I decided that I'd rather have this PR in :) So I reverted my change and we can discuss that issue when it comes up (if ever)

**Comment by sappelhoff** [comment]
thanks @hoechenberger

**Pull request merged**